packages/tables/src/Concerns/CanSelectRecords.php

```diff
@@ -45,6 +45,14 @@ public function getAllTableRecordsCount(): int
             return $this->records->total();
         }
 
+        if ($this->isTableSelectionEnabled() && is_callable($this->getTableRecordBulkActionsEnabled())) {
```

Comment:
I don't think it's necessary to check `isTableSelectionEnabled` since the count is only needed when selection is enabled.

```suggestion
        if (is_callable($this->getTableRecordBulkActionsEnabled())) {
```